### PR TITLE
Create and Populate Database

### PR DESCRIPTION
**Pull request by HannahTierneyV**
The createRecipeDb SQL script creates the following tables and populates them with initial values:
- Recipe
- Ingredient
- RecipeIngredient
- Step
- RecipeStep
- User
- UserRecipe
- UserFavorite

I tested the script by importing it into phpMyAdmin and it works for me, but I do recommend testing it yourself first just to be safe.

### CODE REVIEW

File: createRecipeDb.sql [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+    `recipe_id`     INT     NOT NULL,
+    PRIMARY KEY (`id`),
+    FOREIGN KEY (`user_id`) REFERENCES `User` (`id`),
+    FOREIGN KEY (`recipe_id`) REFERENCES `Recipe` (`id`)
+);
+
+--
+-- Populate `UserFavorite` table with initial values
+--
+INSERT INTO `UserFavorite` VALUES
```
Comment (jctrotter) [inline createRecipeDb.sql]: This bit doesn't work for me when bulk executing the SQL, but works when executing individually. Might just be doing something wrong on my end.

Comment (HannahTierneyV) [reply createRecipeDb.sql]: I'll take a look at it and try replicating the issue.

**Comment by jctrotter** [comment]
Merge into dev instead of master

**Comment by HannahTierneyV** [comment]
Updated the base from master to dev! Thanks for catching that; I forgot to update this PR when you added the dev branch.

**Pull request merged**